examples/kafka/README.md

```diff
@@ -0,0 +1,76 @@
+How to run kafka on PAI
+===
+If you just want to make a very simple example to see whether you can use kafka on PAI, you could submit the job with the kafka.json file in this folder. Then, you can see the stdout in the "Go to Tracking Page" page. You could see that the program has already producerd and consumed message with kafka. 
+
+And we have prepared a docker image with zookeeper, kafka and kafka-python for you, you can use it by build image with the "Dockerfile.example.kafka" file:`sudo docker build -f Dockerfile.example.kafka -t kafka.test .`. And then "run" and "exec" it!
+
+But, if you want to use kafka by yourself, you should follow the next steps.
+
+Please follow the next process step by step. Do not skip any steps if you don't know whether you can skip it.
+
+Zookeeper
+---
+If you already have a kafka cluster or build the image by "Dockerfile.example.kafka", you could skip this step.
+
+1.[Download zookeeper source file](https://zookeeper.apache.org/doc/r3.1.2/zookeeperStarted.html) and unpack it.
+
+2.Change the name of file "/conf/zoo_sample.cfg" to "/conf/zoo.cfg" and open it.
+
+3.Set the "dataDir" and "clientPort" like "dataDir=/var/zookeeper clientPort=2181". Remember the port you set.
+
+4.Start it `bin/zkServer.sh start`.
+
+Kafka
+---
+If you already have a kafka cluster or build the image by "Dockerfile.example.kafka", you could skip this step.
+
+Just take reference in the [official document](https://www.tutorialspoint.com/apache_kafka/apache_kafka_installation_steps.html).
+
+Note:
+
+1.You should set the listener in config/server.properties:"listeners=PLAINTEXT://localhost:9092". The port "9092" is the kafka client port you should use in your client code.
```

Comment:
>9092 [](start = 91, length = 4)

please do not hardcode any specific port number in readme. port conflicts may happen

Reply:
I think you need a config.sh screen to change the configuration according to PAI runtime variables.

---
In reply to: [198105466](https://github.com/Microsoft/pai/pull/771#discussion_r198105466) [](ancestors = 198105466)